src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -702,9 +709,14 @@ private TexlNode ParseExpr(Precedence precMin, TexlNode node = null)
                             if (node is not FirstNameNode first || first.Ident.AtToken != null || _curs.TidPeek() != TokKind.At)
                             {
                                 goto default;
-                            }
-
-                            node = ParseScopeField(first);
+                            }
+                            
+                            if (_features.HasFlag(Features.DisableRowScopeDisambiguationSyntax))
+                            {
+                                PostError(_curs.TokCur, errKey: TexlStrings.ErrBadToken);

```

Comment:
consider a more specific error message that this feature is deprecated/not supported.